src/plot_api/plot_api.js

```diff
@@ -1946,7 +1946,7 @@ function _relayout(gd, aobj) {
 
                 // clear polar view initial stash for radial range so that
                 // value get recomputed in correct units
-                if(fullLayout._subplots.polar && fullLayout[p.parts[0]]) {
+                if(Array.isArray(fullLayout._subplots.polar) && fullLayout._subplots.polar.length && fullLayout[p.parts[0]]) {
```

Comment:
Oh, I didn't notice this logic before - should it be further restricted to when the axis (I guess this is `p.parts[1]`) is `radialaxis`?

Reply:
Good call -> https://github.com/plotly/plotly.js/pull/2200/commits/473643cf8dfb12f1543bd8dcb01ca00c568f6398